### PR TITLE
Update TOS in login

### PR DESCRIPTION
**Pull request by scruffian**
This pull request as TOS text to our login page, inline with our terms of service
  
screen shot 2017-07-20 at 16 54 43
  
#### Testing instructions
  
1. Run `git checkout update/tos` and start your server
2. Open the [`Login` page](http://calypso.localhost:3000/log-in)
3. Check that you see the TOS text above the Log In button
  
#### Additional notes
This exposes a limitation with `preventWidows`, when used strings that contain a link,  but that can be fixed in a different PR.
  
#### Reviews
  
- [x] Code
- [x] Product

### CODE REVIEW

**Comment by breezyskies** [comment]
Minor note, but might be good for the full "Terms of Service" link to break to the next line so it's not so awkward.

**Comment by scruffian** [comment]
> Minor note, but might be good for the full "Terms of Service" link to break to the next line so it's not so awkward.

Yeah I tried to do this, but because of the issue with `preventWidows` it doesn't work.

**Comment by breezyskies** [comment]
@scruffian Pushed a commit that does this with CSS instead. 

Product 👍

File: client/blocks/login/style.scss [hunk 1]
```diff
@@ -54,6 +54,15 @@
 	}
 }
 
+.login__form-terms {
+	font-size: 11px;
+	text-align: center;
+
+	a {
+		white-space: pre;
```
Comment (scruffian) [inline client/blocks/login/style.scss]: Learnt something today 👍

**Comment by scruffian** [comment]
Good job @breezyskies 
Code 👍

**Pull request merged**